data_transfer/lib/byteflies.py

```diff
@@ -96,32 +100,38 @@ def get(self, signal_id: str, algorithm_id: str = "") -> Any:
 
 
 def download_file(
-    download_folder: Path,
     session: requests.Session,
+    download_folder: Path,
     studysite_id: str,
     recording_id: str,
     signal_id: str,
     algorithm_id: str = "",
-) -> bool:
+) -> Union[bool, str]:
```

Comment:
This varied return type seems unsual? Maybe `Optional[str]` would be more explict in the intended outcome, i.e., a filename?

Reply:
Happy to do that, was just following the Dreem implementation (which returns a `bool`)